test/AWSUtils.Tests/Streaming/SQSStreamTests.cs

```diff
@@ -70,7 +62,17 @@ public void Configure(ISiloHostBuilder hostBuilder)
                     .AddSqsStreams("SQSProvider2", options =>
                      {
                          options.ConnectionString = AWSTestConstants.DefaultSQSConnectionString;
-                     });
+                     })
+                    .AddDynamoDBGrainStorage("DynamoDBStore", options =>
+                    {
+                        options.SecretKey = $"Service={AWSTestConstants.Service}";
```

Comment:
@galvesribeiro, am I setting these options correctly?  I this where they need be set, I just don't know which options to set..

Reply:
Sorry, mislooked that part. Just need to set `options.Service` to the `AWSTestConstants.Service`. That property has the emulator URL already there. https://github.com/jason-bragg/orleans/blob/c1a19f8ced75735db95ecd1ecf8d856ade42fc3a/test/AWSUtils.Tests/StorageTests/AWSTestConstants.cs#L45-L47